src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,141 @@
+import _ from 'lodash';
+import React from 'react';
+import PropTypes from 'react-peek/prop-types';
+import { lucidClassNames } from '../../util/style-helpers';
+import { buildHybridComponent } from '../../util/state-management';
+import {
+	ExpanderPanelDumb as ExpanderPanel,
+	IExpanderPanelHeaderProps,
+} from '../ExpanderPanel/ExpanderPanel';
+import {
+	findTypes,
+	omitProps,
+	StandardProps,
+	Overwrite,
+} from '../../util/component-types';
+import * as reducers from '../Accordion/Accordion.reducers';
+
+const cx = lucidClassNames.bind('&-Accordion');
+
+const { any, func, object, number, string } = PropTypes;
+
+interface IAccordionPropsRaw extends StandardProps {
+	/**
+	 * Indicates which item is expanded
+	 * */
+	selectedIndex: number;
+	/**
+	 * Called when the user clicks on the component's header of an item.
+	 * */
+
+	onSelect: (
+		selectedIndex: number | null,
+		{ event, props }: { event: React.MouseEvent; props: IAccordionProps }
+	) => void;
+	/**
+	 * Prop alternative to Header child component passed through to the
+	 * underlying ExpanderPanel
+	 */
+	Header: IExpanderPanelHeaderProps;
+}
+
+type IAccordionProps = Overwrite<
+	React.DetailedHTMLProps<React.HTMLAttributes<HTMLDivElement>, HTMLDivElement>,
+	IAccordionPropsRaw
+>;
```

Comment:
Do we need to use Overwrite here? Or can we just name the above interface `IAccordionProps`?

Reply:
I used Overwrite because `onSelect` in `IAccordionPropsRaw` conflicts with `React.DetailedHTMLProps...`. It seems like we aren't even using `React.DetailedHTMLProps...` in other class components, so I'm guessing I don't actually need to extend it and can just use `IAccordionPropsRaw` as ``IAccordionProps` like you said.

Reply:
Oh yea, I think you are right to use it then. Otherwise we'll get some browser errors due to conflicting signatures for onSelect. Sorry for the confusion.